openedx/core/djangoapps/credit/urls.py

```diff
@@ -1,43 +1,53 @@
 """
 URLs for the credit app.
 """
-from django.conf.urls import patterns, url
+from django.conf.urls import patterns, url, include
+from rest_framework.routers import DefaultRouter
 
-from .api.provider import get_credit_provider_info
-from .views import create_credit_request, credit_provider_callback, get_providers_detail, get_eligibility_for_user
+from openedx.core.djangoapps.credit import views
+from openedx.core.djangoapps.credit.api.provider import get_credit_provider_info
 
 PROVIDER_ID_PATTERN = r'(?P<provider_id>[^/]+)'
 
-urlpatterns = patterns(
+V1_URLS = patterns(
     '',
 
     url(
-        r"^v1/providers/(?P<provider_id>[^/]+)/$",
```

Comment:
Are the old URLs used by anyone? why are they being deleted?  Did these URLs go out with Cypress?   Do we need to retain them in a deprecated state for one release so that we don't break future releases to the open source community?